src/database/GameInstance.ts

```diff
@@ -28,6 +28,22 @@ export class GameInstance {
     this.settings = {};
   }
 
+  public reset() {
+    this.gameId = undefined;
+    this.finished = undefined;
+    this.announced = false;
+    this.startTime = undefined;
+    this.endTime = undefined;
+    this.settings = {
+      minerushing: undefined,
+      bannedClasses: undefined,
+      map: undefined,
+    };
+    this.teams = { RED: [], BLUE: [] };
+    this.mapVoteManager = undefined;
+    this.minerushVoteManager = undefined;
+  }
+
```

Comment:
how about when the game ends or whatever, `CurrentGameManager` calls `GameInstance.commitToDb()` or smth
and calls `this.currentGame = new GameInstance()`

Reply:
add a note to do this in the new `resetGameInstance()` method